src/share/file.ts

```diff
@@ -81,39 +80,23 @@ export const getLatestLogFile = async (dir: string) => {
   }
 };
 
-const authReg = /https:\/\/riot:.+@127\.0\.0\.1:\d+\/index.html/
 
 export const getLcuToken = async (dirPath: string) => {
   const appendGameToDir = config.get(`appendGameToDir`);
-  const dir = `${appendGameToDir ? `${dirPath}/Game` : dirPath}/Logs/LeagueClient Logs`;
+  const lockfilePath = `${appendGameToDir ? `${dirPath}/LeagueClient` : dirPath}/lockfile`;
 
   try {
-    const files = await fs.readdir(dir);
-    const rendererLogs = files
-      .filter((f: string) => f.includes(`renderer.log`))
-      .sort((a: string, b: string) => b.localeCompare(a));
-
-    let content = ''
-    for(const f of rendererLogs) {
-      content = await fs.readFile(`${dir}/${f}`, 'utf8');
-      if (authReg.test(content)) {
-        break
-      }
-      continue
-    }
-
-    if (!content) {
-      console.error(`[LCU] cannot find target renderer log.`)
-      return [null, null, null]
-    }
-
-    const url = content.match(/https(.*)\/index\.html/)?.[1] ?? ``;
-    const token = url.match(/riot:(.*)@/)?.[1] ?? null;
-    const port = url.match(/:(\d+)$/)?.[1] ?? null;
+    const lockfile = await fs.readFile(`${lockfilePath}`, 'utf8');
+    const port = lockfile.split(`:`)[2];
+    const token = lockfile.split(`:`)[3];
+
+    const url = `://riot:${passwd}@127.0.0.1:${port}`
```

Comment:
Looks like this one was forgot. https://github.com/cangzhang/champ-r/runs/3552301364

Reply:
thanks for reviewing 🥵